python/ray/serve/_private/logging_utils.py

```diff
@@ -225,8 +227,44 @@ def record_factory(*args, **kwargs):
     logger.addHandler(file_handler)
 
 
+def configure_component_memory_logger(
```

Comment:
nit: `memory_profiler` rather than logger

Reply:
Sounds good, I renamed it.